src/idiom314ex.fz

```diff
@@ -0,0 +1,12 @@
+ex314 is
+
+  # Arrays in Fuzion are immutable, creating a new array instead
+
+  x := [1,2,3,4,5]
+  v := 0
+
+  x1 := (v :: x->x).take x.length
```

Comment:
I think this might be confusing. I would propose to just do `x1 := x.map_to_array _->v`